core/src/test/scala/org/apache/spark/scheduler/TaskContextSuite.scala

```diff
@@ -92,7 +92,7 @@ class TaskContextSuite extends SparkFunSuite with BeforeAndAfter with LocalSpark
       0, 0, taskBinary, rdd.partitions(0), Seq.empty, 0, new Properties,
       closureSerializer.serialize(TaskMetrics.registered).array())
     intercept[RuntimeException] {
-      task.run(0, 0, null, null, Option.empty)
+      task.run(0, 0, null, 0, null, Option.empty)
```

Comment:
it would be nice to have a test with stage level scheduling included, perhaps look at SparkContextSuite, also having a value other then 1 would be good.

Reply:
Added a test in SparkContextSuite